open-api/rest-catalog-open-api.yaml

```diff
@@ -1559,6 +1578,22 @@ components:
             type: string
           description:
             Properties that should be used as default configuration; applied before client configuration.
+        capabilities:
+          type: array
+          uniqueItems: true
+          items:
+            type: string
+            enum:
+              - pagination
+              - scan-planning
+              - views
+              - vended-credentials
```

Comment:
To match others, I think this should be `credential-vending`.

Reply:
I specifically tried to match https://github.com/apache/iceberg/blob/99f81437432c8bf24ab7fc06e30ad87d243e5ad5/open-api/rest-catalog-open-api.yaml#L1506